.github/CODEOWNERS

```diff
@@ -16,8 +17,81 @@
 # In each subsection folders are order first by depth, then alphabetically.
 # This should make it easy to add new rules without breaking existing ones.
 
+# Area owners of the dotnet/runtime repo: https://github.com/dotnet/runtime/blob/master/docs/area-owners.md 
+
+/xml/System.Buffers**/** @layomia @JeremyKuhne
+/xml/System.CodeDom**/** @buyaa-n @krwq
+/xml/System.Collections**/** @eiriktsarpalis @layomia
+/xml/System.ComponentModel**/** @JeremyKuhne @safern
+/xml/System.ComponentModel.Composition**/** @maryamariyan @ViktorHofer
+/xml/System.ComponentModel.DataAnnotations**/** @lajones @ajcvickers
+/xml/System.Composition**/** @maryamariyan @ViktorHofer
+/xml/System.Configuration**/** @maryamariyan @safern
+/xml/System.Data**/** @ajcvickers @cheenamalhotra @david-engel
+/xml/System.Data.SqlClient**/** @cheenamalhotra @david-engel @karinazhou @JRahnama
+/xml/System.Diagnostics**/** @tommcdon @krwq
+/xml/System.Diagnostics/DiagnosticSource.xml @Anipik
+/xml/System.Diagnostics/EventLog*.xml @Anipik
+/xml/System.Diagnostics/PerformanceCounter*.xml @Anipik
+/xml/System.Diagnostics/Process*.xml @adsitnik @eiriktsarpalis
+/xml/System.Diagnostics/TraceSource.xml @Anipik
+/xml/System.Diagnostics.Tracing**/** @Anipik
+/xml/System.DirectoryServices**/** @tquerec @josephisenhour
+/xml/System.Drawing**/** @JeremyKuhne @maryamariyan
+/xml/System.Dynamic**/** @cston @333fred
+/xml/System.Globalization**/** @safern @krwq @tarekgh
+/xml/System.IO.Compression**/** @carlossanlop @ericstj
+/xml/System.IO.Pipelines**/** @davidfowl @halter73 @jkotalik @anurse
+/xml/System.IO**/** @JeremyKuhne @carlossanlop @jozkee
```

Comment:
This line should move up above line 44. As written, this rule will override both of the previous rules.

Reply:
Fixed